test/Liminality.Tests/VisualizationTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using PSIBR.Liminality.Tests.Fixtures;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PSIBR.Liminality
+{
+    public class VisualizationTests : IClassFixture<BasicStateMachineFixture>
```

Comment:
Things I found:

- Nested states/signals should be represented with DeclaringType.NestedType.FinalType name format
-  re-entrant states arent displaying.